src/index.js

```diff
@@ -38,51 +40,38 @@ addParseCloud();
 // "dotNetKey": optional key from Parse dashboard
 // "restAPIKey": optional key from Parse dashboard
 // "javascriptKey": optional key from Parse dashboard
+
 function ParseServer(args) {
-  if (!args.appId || !args.masterKey) {
-    throw 'You must provide an appId and masterKey!';
-  }
+  // Setup providers
+  CacheProvider.setup(args.cache);
+  FilesProvider.setup(args.files);
+  DatabaseProvider.setup(args.database);
+
+  // Instantiate the app
+  var app = new ParseApp(args.app);
 
-  if (args.databaseAdapter) {
-    DatabaseAdapter.setAdapter(args.databaseAdapter);
-  }
-  if (args.filesAdapter) {
-    FilesAdapter.setAdapter(args.filesAdapter);
-  }
-  if (args.databaseURI) {
-    DatabaseAdapter.setAppDatabaseURI(args.appId, args.databaseURI);
-  }
   if (args.cloud) {
+    // Add the Parse.Cloud global function definitions
     addParseCloud();
-    if (typeof args.cloud === 'function') {
-      args.cloud(Parse)
-    } else if (typeof args.cloud === 'string') {
-      require(args.cloud);
+
+    // Load the cloud code entry point
+    if (typeof args.cloud.entry === 'function') {
```

Comment:
Just semantics like you say, implemented as well cloud code options here: https://github.com/ParsePlatform/parse-server/pull/263/files#diff-1fdf421c05c1140f6d71444ea2b27638R53